src/svc/cloud-data-sources/paths/orgs-orgID-datasources.yml

```
@@ -0,0 +1,40 @@
+    get:
+      summary: ''
+      operationId: get-orgs-orgID-datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: ./svc/cloud-data-sources/schemas/Datasource.yml
+      description: Get a list of datasources for this organization
+    post:
+      summary: ''
+      operationId: post-orgs-orgID-datasources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                  endpoint:
+                    $ref: '../schemas/ListenerEndpoint'
```

This file has an extension
```suggestion
                    $ref: '../schemas/ListenerEndpoint.yml'
```